ElectoralRegisterResidentInformationApi.Tests/V1/UseCase/GetAllUseCaseTests.cs

```diff
@@ -1,26 +1,26 @@
 using System.Linq;
 using AutoFixture;
-using BaseApi.V1.Boundary.Response;
-using BaseApi.V1.Domain;
-using BaseApi.V1.Factories;
-using BaseApi.V1.Gateways;
-using BaseApi.V1.UseCase;
+using ElectoralRegisterResidentInformationApi.V1.Boundary.Response;
+using ElectoralRegisterResidentInformationApi.V1.Domain;
+using ElectoralRegisterResidentInformationApi.V1.Factories;
+using ElectoralRegisterResidentInformationApi.V1.Gateways;
+using ElectoralRegisterResidentInformationApi.V1.UseCase;
 using FluentAssertions;
 using Moq;
 using NUnit.Framework;
 
-namespace BaseApi.Tests.V1.UseCase
+namespace ElectoralRegisterResidentInformationApi.Tests.V1.UseCase
 {
     public class GetAllUseCaseTests
     {
-        private Mock<IExampleGateway> _mockGateway;
+        private Mock<IElectoralRegisterResidentInformationApiGateway> _mockGateway;
```

Comment:
See above. I think the gateway and its interface don't need to have `ResidentInformationApi` in the name.